### PR TITLE
fix UTF code formatting

### PR DESCRIPTION
**Pull request by BillWagner**
The Euro characters were not formatted correctly.

This PR fixes it.

[Internal review link](https://review.docs.microsoft.com/en-us/dotnet/standard/parallel-programming/task-based-asynchronous-programming?branch=pr-en-us-3821)

cc @richlander

### CODE REVIEW

**Comment by mairaw** [comment]
I'm trying to find another example where I did a similar change to see if I also changed the file encoding. @rpetrusha do you recall that? We should probably check our examples that have Euro symbol or mention euro for example to see if they're all displaying correctly.

**Comment by mairaw** [comment]
Found it: #3113. But in that case, the file was not UTF-8. Should we be seeing � in the code?

**Comment by richlander** [comment]
Thanks for fixing it.

**Comment by rpetrusha** [comment]
For UTF8, BOM, as I recall, isn't actually used; it's probably best to save without BOM. 
The problem goes well beyond the Euro symbol, though. There are numerous examples with non-ASCII characters and character sets, all of which are affected if files aren't saved in UTF-8 format. So I'm wondering how extensive this encoding issue is, and what it's cause is. I may have been the source of the error, though I thought that I had configured my text editors to automatically save all source code files as UTF-8. But it's also possible that the issue arose during one of our migrations. 
We can check for the Euro symbol by looking for something like a ":C2" format string, which will catch 
some but not all occurrences. But I think it's much harder to find other non-ASCII characters.
//cc @mairaw @BillWagner

**Comment by mairaw** [comment]
Ok. So maybe in that example the file was in a different encoding. Let’s open an issue to look at this globally and try to think of examples where this could be an issue. Could be an up-for-grabs, P2 kind of thing.

**Comment by BillWagner** [comment]
I agree with @mairaw's proposal. I tried some ideas to search for UTF characters that weren't displayed correctly in the repo, and I couldn't come up with a good search string. It may be that spotting them by hand will be the best solution 😭 

Let's merge this one, and write the new issue.

**Review by rpetrusha — APPROVED** [review]
@BillWagner, the example that you fixed is also used in the [CultureInfo class](https://review.docs.microsoft.com/en-us/dotnet/api/system.globalization.cultureinfo?view=netframework-4.7.1&branch=pr-en-us-3821#Async), where it now is correctly formatted. But the other three examples found in the same directory are all stored as ASCII files. Do you want to change those as part of this PR, or do you want to defer it as part of the larger effort reflected in the issue? I'll approve this PR and leave it up to you to decide.

**Comment by BillWagner** [comment]
@rpetrusha  I just fixed the other files.  I'll do a sanity check once the build finishes and then merge.

**Pull request merged**
